app/assets/stylesheets/shared/question-mark-icon.scss

```diff
@@ -58,6 +56,7 @@
   margin-left: -7.4rem;
   margin-top: -0.1rem;
   background-color: transparent;
+  z-index: 1005;
```

Comment:
is there any science behind this number? I assume so

Reply:
Yes, modal is `1004`
So, it 1004 + 1

Reply:
any chance we could better show our intent with vars?

Reply:
Yes, sure.
See commit d0e123f00